maindataclasses.rb

```diff
@@ -1,22 +1,33 @@
 # task 1
+puts "task1"
 string1 = '235'
-sum = string1[0].to_i+string1[1].to_i+string1[2].to_i
+v = 1
+len = string1.length
+sum = string1[0].to_i
+while v < len
+  sum += string1[v].to_i
+  v += 1
+end
 puts sum
 
 # task 2
+puts "task2"
 string2 = 'afsdf3432df23dj334'
 array = string2.scan(/[0-999]+/)
 puts array.max_by{ |elem| elem.size }.length
 
+
 # task 3
+puts "task3"
 string2 = 'afsdf3432df23dj334'
 array = string2.scan(/[0-999]+/)
 puts array.max_by{ |elem| elem.size }
```

Comment:
you were asked to find a maximal number (not by size but value)
111 != 999 for example